internal/datalakes/inmemory/policyresolver.go

```diff
@@ -615,6 +615,19 @@ func (db *Db) SetResolvedPolicy(ctx context.Context, mrn string, resolvedPolicy
 		return errors.New("failed to save resolved policy '" + mrn + "'")
 	}
 
+	// We need the risk factors for initial reporting, but don't require all
+	// their metadata. The risk factors in the resolved policy provides everything
+	// we need for scoring. If we fetch a full risk factor we can replace it.
+	for mrn, rf := range resolvedPolicy.CollectorJob.RiskFactors {
```

Comment:
"For any later reporting step we may still inject all risk factor metadata into the inmemory store before printing the output."
i.e. since we only pull from the resolved policy here it only has super limited metadata. Whenever we can we want to pull from the bundle. Just in this case, we don't get a bundle until we start printing - or not at all if we are in serve mode.

Reply:
Thanks for considering it!! could have easily been a miss on my part